src/test/cypress/use-cases.yaml

```diff
@@ -0,0 +1,53 @@
+pool:
+  name: altinn-vmss-testagent
+
+name: $(environment)_Studio_Usecases_$(Date:ddMMyyyy)_$(Date:HHmm)
+
+trigger: none
+
+pr: none
+
+variables:  
+  cyCacheDir: "/home/AzDevOps/.cache/Cypress"
+  cypressTestDir: "$(Build.SourcesDirectory)/src/test/cypress"
+
+schedules:
+- cron: "10,25,40,55 * * * *"
+  displayName: Bruksmønster
+  branches:
+    include:
+    - master
+  always: true
+
+steps:
+- task: DeleteFiles@1
+  displayName: 'Delete test results'
+  inputs:
+    SourceFolder: '$(cypressTestDir)/reports'
+    Contents: |
+      *.xml
+  condition: succeededOrFailed()
+
+- bash: |
+   yarn --immutable
+
+   npx cypress cache list
```

Comment:
What does `npx` do here? Is cypress not already installed on the agent? Is this necesarry?

Also `npx ` is an npm-command.  If we need this command could we use the yarn alternative `yarn dlx cypress` .

Reply:
with `dlx`, the script took longer time. so moved the command to `package.json` and using `yarn run` in yaml.